src/data/set/pointwise.lean

```diff
@@ -470,6 +471,29 @@ begin
     exact ih.trans (subset_mul_right _ hs) }
 end
 
+lemma mem_pow : ∀ {a : α} {n : ℕ} (hn : n ≠ 0),
```

Comment:
```suggestion
@[to_additive] lemma mem_pow : ∀ {a : α} {n : ℕ} (hn : n ≠ 0),
```
and you're missing the `finset` and `filter` versions.

Reply:
Whoops, the n=0 case was left from similar work on submodule where it's not true...

Reply:
I added the `finset` case, I don't think the surrounding lemmas exist for `filter`